tests/test_display.py

```diff
@@ -23,24 +23,65 @@ def test_symbols():
     # This test has no assertions, except from the human eye. When it fails,
     # ticks and crosses fail and must be inspected by the human eye
     if console.has_unicode():
-        symbols = display.unicode_symbols
+        symbol_groups = (display.html_symbols, display.unicode_symbols, display.ascii_symbols)
     else:
-        symbols = display.ascii_symbols
-    print("[{0}][{1}][{2}][{3}]".format(
-        symbols[py_trees.common.Status.SUCCESS],
-        symbols[py_trees.common.Status.FAILURE],
-        symbols[py_trees.common.Status.INVALID],
-        symbols[py_trees.common.Status.RUNNING]
+        symbol_groups = (display.html_symbols, display.ascii_symbols)
+
+    for symbols in symbol_groups:
+
+        print("Status: [{0}][{1}][{2}][{3}]".format(
+            symbols[py_trees.common.Status.SUCCESS],
+            symbols[py_trees.common.Status.FAILURE],
+            symbols[py_trees.common.Status.INVALID],
+            symbols[py_trees.common.Status.RUNNING]
+            )
         )
-    )
 
-    print("[{0}][{1}][{2}][{3}]".format(
-        symbols[py_trees.behaviours.Behaviour],
-        symbols[py_trees.composites.Sequence],
-        symbols[py_trees.composites.Selector],
-        symbols[py_trees.composites.Parallel]
+        print("Classes: [{0}][{1}][{2}][{3}]".format(
+            symbols[py_trees.behaviours.Behaviour],
+            symbols[py_trees.composites.Sequence],
+            symbols[py_trees.composites.Selector],
+            symbols[py_trees.composites.Parallel]
+            )
+        )
+
+
+def test_html_tree():
+    console.banner("Ascii/Html Snapshots - Comparison Check")
+
+    def print_tree(snapshot_visitor, tree):
```

Comment:
Probably can try parsing the data as xml instead of just printing it.

Reply:
Good idea, now parses the xml and it helped ensure it's now a valid xhtml snippet, not just html (problems with &nbsp;)